src/domain/entities/LandingNode.ts

```diff
@@ -92,6 +96,25 @@ export const updateLandingNodes = (
         })
         .compact()
         .value();
+
+    return updatedNodes.map(node => applyFavicon(node));
+};
+
+const applyFavicon = (parent: LandingNode): LandingNode => {
+    const spreadFaviconToChildren = (children: LandingNode[], favicon: string): LandingNode[] => {
+        return _.map(children, child => {
+            const updatedChild: LandingNode = { ...child, favicon };
+            if (child.children) {
+                updatedChild.children = spreadFaviconToChildren(child.children, favicon);
+            }
+            return updatedChild;
+        });
+    };
+
+    return {
+        ...parent,
+        children: parent.children ? spreadFaviconToChildren(parent.children, parent.favicon) : [],
```

Comment:
```
Boolean([]) // true
```

No need to have this ternary operator (it even made me doubt if in runtime it could be an undefined, but checked [buildLandingNode](https://github.com/EyeSeeTea/home-page-app-dev/blob/a82d9b60948882e38a8ae025b126f813919d3717/src/data/repositories/LandingNodeDefaultRepository.ts#L126))